client/app/services/query.js

```diff
@@ -52,7 +52,7 @@ export class Query {
       this.options = {};
     }
     this.options.apply_auto_limit = !!this.options.apply_auto_limit;
-    this.options.apply_long_query = false;
+    this.options.apply_long_query = !!this.options.apply_long_query;
```

Comment:
This was not the right solution to the issue. This changed the `Mark as Long Query` checkbox to unchecked every time we opened the query. 

To default the `Mark as Long Query` checkbox for new queries to unchecked, the below change is made.